webapp/portlet/src/main/webapp/vue-apps/notification-top-bar/components/ExoTopBarNotification.vue

```diff
@@ -11,7 +11,7 @@
             :value="badge > 0"
             :content="badge"
             flat
-            color="var(--allPagesBadgePrimaryColor, #ff5335)"
+            color="var(--allPagesBadgePrimaryColor, #D32A2A)"
```

Comment:
```suggestion
            color="var(--allPagesBadgePrimaryColor, #d32a2a)"
```
can you use the variable instead ?